packages/voorbeeld-design-tokens/figma/voorbeeld.tokens.json

```diff
@@ -930,7 +945,7 @@
         },
         "font-size": {
           "$type": "fontSizes",
-          "$value": "{voorbeeld.typography.font-size.md}"
+          "$value": "{basis.typography.scale.medium}"
```

Comment:
Heb je het getest in Storybook enzo? Ik denk dat dit `{basis.typography.scale.medium.font-size}` moet zijn.

Als je klaar bent, zet de Pull Request dan van "draft" naar "ready", zodat de Chromatic visuele regressietests aan de slag gaan.

Reply:
Ik had het geprobeerd en het was nog steeds undefined, ik miste ook volgensmij een dollar sign. Maar nu gefixt, ze zijn er beide